edgexfoundry/edgex-go
# refactor!: Change configuration file format to YAML

BREAKING CHANGE: Configuration files are now in YAML format, Default file name is now configuration.yaml

closes # 4452







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
run `make docker`
from compose builder run `make run delayed-start dev`
Verify all core/support/security services bootstrap successfully and contain the following log message:
```
msg="Loaded private configuration from ./res/configuration.yaml"
```

## New Dependency Instructions (If applicable)
